geth/api/backend.go

```diff
@@ -273,13 +264,28 @@ func (b *StatusBackend) registerHandlers() error {
 	if rpcClient == nil {
 		return node.ErrRPCClient
 	}
-	rpcClient.RegisterHandler("eth_accounts", func(context.Context, ...interface{}) (interface{}, error) {
+	rpcClient.RegisterHandler(params.PersonalSignMethodName, func(context.Context, ...interface{}) (interface{}, error) {
+		return nil, errors.New("IGORM: custom erro boi from an override!")
```

Comment:
IGORM in an error description? :) Also, `erroboi` is a typo?
Maybe just "TBD" or "TODO"?

Reply:
Lol, missed that one! Thanks!